calculator.css

```diff
@@ -0,0 +1,7 @@
+#Name {
+    font-weight: bold;
+}
+
+#Name {
+    font-size:50px;
+}
```

Comment:
you can put both of these under #Name like so:
#Name {
    font-weight: bold;
    font-size:50px;
}
Also, please use camelCase (first letter undercase, all subsequent words uppercase)